chat-plugins/trivia.js

```diff
@@ -535,11 +535,9 @@ let Trivia = (function () {
 		let participantsLen = this.participants.size;
 		if (!participantsLen) return output.sendReplyBox("There are no players in this trivia game.");
 
-		let participants = [];
 		let buffer = "There " + (participantsLen > 1 ? "are <strong>" + participantsLen + "</strong> players" : "is <strong>1</strong> player") + " participating in this trivia game:<br />";
-		this.participants.forEach(function (scoreData, participantid) {
-			let participant = Users(participantid);
-			participants.push(participant ? participant.name : participantid);
+		let participants = Array.from(this.participants.keys(), function(participantid) {
```

Comment:
When @Zarel realises that using them in non-critical code paths isn't a statically significant performance loss